jsapp/js/components/processing/singleProcessingStore.ts

```diff
@@ -653,22 +658,22 @@ class SingleProcessingStore extends Reflux.Store {
     this.trigger(this.data);
   }
 
-  private isAutoTranscriptionEventApplicable(event: AutoTranscriptionEvent) {
+  private isAutoTranscriptionEventApplicable(event: AutoTransXEvent) {
     // Note: previously initiated automatic transcriptions may no longer be
     // applicable to the current route
-    const googleTsResponse =
-      event.response[this.currentQuestionQpath]?.googlets;
+    const googleTxResponse =
+      event.response[this.currentQuestionQpath]?.googletx;
```

Comment:
@Guitlle shouldn't this be `.googlets` and `googleTsResponse`?

Reply:
Yes, this is the missing part for the frontend, it's still WIP for the frontend, maybe we can talk about it later, I think I could handle it but I have some questions